include/labwc.h

```diff
@@ -107,6 +107,9 @@ struct seat {
 	enum lab_cursors server_cursor;
 	struct wlr_cursor *cursor;
 	struct wlr_xcursor_manager *xcursor_manager;
+	struct {
+		double x, y;
```

Comment:
Do we really need both here? Or, in other words, could we receive / should we handle two scrolling actions on different axis at once?

Reply:
Yes; when a user scrolls "diagonally" both axes have nonzero deltas interleaved.